src/components/header/Backtrace.css

```diff
@@ -0,0 +1,25 @@
+.backtrace {
+  margin: 5px;
+  padding: 0;
+  list-style: none;
+  display: block;
```

Comment:
Wow, I never knew `<ol>` was an inline element !

Can you please move this line at the top of the rule, so that it's easier to read ?

Reply:
It's `display:block` by default, what I did here was unnecessary. I'll remove it. Thanks for catching it.